ensembl_dbcopy/migrations/0004_auto_20210129_1245.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.13 on 2021-01-29 12:45
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
I thought the PR was  only for the new field "active" I see some updates on `group` field. Is this intended? or a mix between two concurrent devs?

Reply:
I did the migration while working for groups so it picked those changes . i will redo migrations

Reply:
I recreated the migration file and i see the all the changes which are not related to changes i made .

I think these are previous migrations not applied  : 

https://github.com/Ensembl/ensembl-production-services/pull/115/files#diff-45432eec4270f212866cd5050af5fb0b139ab76887c4989eddac88b0f4ff31eeR25 

https://github.com/Ensembl/ensembl-production-services/pull/115/files#diff-45432eec4270f212866cd5050af5fb0b139ab76887c4989eddac88b0f4ff31eeR30